server/models/db/add-worker-timeslot.js

```diff
@@ -0,0 +1,17 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const Tables = require('../../constants/tables.json');
+const AddTimeslotStatus = require('../../constants/timeslot-status.json');
+
+// This method inserts an appointment entry given specific information.
+async function insertWorkerTimeslot(workerTimeslotId, slotId, schoolId, workerId, status, date) {
+    // Added conditional statements to prevent purpose, studentNotes, and workerComments from storing "null" instead of null.    
+    // Assumes all new appointments are upcoming and not for the past (as front end only allows users to book appointments for future dates).
+    return db.any(`insert into ${Tables.worker_timeslot} (worker_timeslot_id, slot_id, school_id, worker_id, status, date) values (${workerTimeslotId}, ${slotId}, ${schoolId}, ${workerId}, ${status ? `'${status}'` : null}, ${date ? `'${date}'` : null}, '${AddTimeslotStatus.available}');`);
```

Comment:
Please note the ambiguity surrounding the date data type in this method. 
I am unsure exactly how to format the date parameter into the statement which inserts the record into the database

Reply:
I would leverage moment to format the date so it is consistent in the database table. Refer to the documentation here: https://momentjs.com/